eva_submission/ingestion_templates.py

```diff
@@ -18,51 +16,46 @@ def accession_props_template(
         mongo_user,
         mongo_pass,
 ):
-    return f"""
-accessioning.instanceId=instance-{instance_id}
-accessioning.submitted.categoryId=ss
-accessioning.monotonic.ss.blockSize=100000
-accessioning.monotonic.ss.blockStartValue=5000000000
-accessioning.monotonic.ss.nextBlockInterval=1000000000
-
-parameters.assemblyAccession={assembly_accession}
-parameters.taxonomyAccession={taxonomy_id}
-parameters.projectAccession={project_accession}
-parameters.chunkSize=100
-parameters.vcf={vcf_path}
-parameters.vcfAggregation={aggregation}
-parameters.forceRestart=false
-parameters.fasta={fasta}
-parameters.outputVcf={output_vcf}
-parameters.assemblyReportUrl=file:{report}
-# contigNaming available values: SEQUENCE_NAME, ASSIGNED_MOLECULE, INSDC, REFSEQ, UCSC, NO_REPLACEMENT
-parameters.contigNaming=NO_REPLACEMENT
-
-spring.batch.job.names=CREATE_SUBSNP_ACCESSION_JOB
-
-spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url={postgres_url}
-spring.datasource.username={postgres_user}
-spring.datasource.password={postgres_pass}
-spring.datasource.tomcat.max-active=3
-spring.jpa.generate-ddl=true
-
-spring.data.mongodb.host={mongo_host}
-spring.data.mongodb.port=27017
-spring.data.mongodb.database=eva_accession_sharded
-spring.data.mongodb.username={mongo_user}
-spring.data.mongodb.password={mongo_pass}
-spring.data.mongodb.authentication-database=admin
-mongodb.read-preference=primaryPreferred
-
-spring.main.web-environment=false
-"""
+    """
+    Get all properties needed for this accessioning job, except for the input
+    and output filenames which are filled in by Nextflow.
+    """
+    return {
+        'accessioning.instanceId': f'instance-{instance_id}',
+        'accessioning.submitted.categoryId': 'ss',
+        'accessioning.monotonic.ss.blockSize': 100000,
+        'accessioning.monotonic.ss.blockStartValue': 5000000000,
+        'accessioning.monotonic.ss.nextBlockInterval': 1000000000,
+        'parameters.assemblyAccession': assembly_accession,
+        'parameters.taxonomyAccession': taxonomy_id,
+        'parameters.projectAccession': project_accession,
+        'parameters.chunkSize': 100,
+        'parameters.vcfAggregation': aggregation,
+        'parameters.forceRestart': False,
+        'parameters.fasta': fasta,
+        'parameters.assemblyReportUrl': f'file:{report}',
+        'parameters.contigNaming': 'NO_REPLACEMENT',
```

Comment:
We should use INSDC replacement by default. Have we loaded any studies using NO_REPLACEMENT thus far?

Reply:
@sundarvenkata-EBI 
The majority of studies have been accessioned using "NO_REPLACEMENT"
The assembly check relies on the contig alias used under the CHROM column to be part of the INSDC assembly report so this is usually the case by the time the VCF gets to this stage.

Reply:
Sorry, I seemed to have jumped the gun here... you are right. With NO_REPLACEMENT, the accessioned VCF output won't  have the INSDC accession if the input VCF didn't have it in the CHROM column but that is what we wanted...